widlparser/constructs.py

```diff
@@ -211,7 +211,7 @@ def peek(cls, tokens: Tokenizer) -> bool:
 						return tokens.pop_position(ConstValue.peek(tokens))
 		return tokens.pop_position(False)
 
-	def __init__(self, tokens: Tokenizer, parent: Construct = None, symbol_table: protocols.SymbolTable = None) -> None:
+	def __init__(self, tokens: Tokenizer, parent: Optional[Construct] = None, symbol_table: protocols.SymbolTable = None) -> None:
```

Comment:
```suggestion
	def __init__(self, tokens: Tokenizer, parent: Construct = None, symbol_table: protocols.SymbolTable = None) -> None:
```
Optional is inferred when there's a None default